lib/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Traditionally we place the "executable" files in a bin/ folder instead of lib/. Also, it's good to add a shebang to them as the first line: `#!/usr/bin/env node`. This makes them directly executable, using `./bin/index.js`.